src/GraphQL/loaders/parseClassQueries.js

```diff
@@ -75,6 +75,62 @@ const load = (parseGraphQLSchema, parseClass) => {
         );
         const parseOrder = order && order.join(',');
 
+        if (where) {
```

Comment:
I am afraid it will not work inside an _or or _and statement. I think you should call _key and _value and use an approach similar to this one: https://github.com/parse-community/parse-server/blob/master/src/GraphQL/loaders/objectsQueries.js#L112

It will even work for generic queries as well.

Reply:
I completely changed my approach to solving this problem, and also added a test that covers _or and _and operators.

Reply:
It may work, but I still think that we should call _key and _value and use the same approach of https://github.com/parse-community/parse-server/blob/master/src/GraphQL/loaders/objectsQueries.js#L112 with the benefit of working also for generic queries and having a less complex code to maintain later.